_locales/_untranslated_en.json

```diff
@@ -29,5 +29,41 @@
   },
   "camera_access_failed": {
     "message": "Could not access video camera. You might want to check your video permissions or if the camera is used already by another program"
+  },
+  "onboarding_create_instant_account": {
+    "message": "Let's get started!"
+  },
+  "onboarding_alternative_logins": {
+    "message": "I already have a login"
+  },
+  "manual_account_setup_option": {
+    "message": "Manual login"
```

Comment:
suggestions from internal chat:

> I think email login under the name "manual Login" it too much hidden / discuoraged, how about calling it "Login with normal/standard email account" or sth like that instead? or "Manual Login with existing email account"

Reply:
let's leave wording/flow discussions outside this PR for now, even if things are not 100% perfect, we need to move forward